client/layout/index.jsx

```diff
@@ -104,7 +105,8 @@ class Layout extends Component {
 				{ 'is-support-session': this.props.isSupportSession },
 				{ 'has-no-sidebar': ! this.props.hasSidebar },
 				{ 'has-chat': this.props.chatIsOpen },
-				{ 'has-no-masterbar': this.props.masterbarIsHidden }
+				{ 'has-no-masterbar': this.props.masterbarIsHidden },
+				{ 'is-jetpack-site': this.props.isJetpack }
```

Comment:
Not introduced by this PR, but the individual `{}` objects are weird. Maybe merge into one?

```
				/* ... */
				{
					'is-support-session': this.props.isSupportSession,
					'has-no-sidebar': ! this.props.hasSidebar,
					'has-chat': this.props.chatIsOpen,
					'has-no-masterbar': this.props.masterbarIsHidden,
					'is-jetpack-site': this.props.isJetpack,
				}
```

Reply:
Yeah, I like your suggestion, but let's do it another time. Some color scheming work is based on that one, so I'd like to keep it as clean as possible 😉